docs/mobile-apps/faq.md

```diff
@@ -96,7 +96,7 @@ Yes. We support OAuth login via Google and GitHub.
 
 #### **Are your devices jailbroken?**
 
-No. We do not offer jailbroken or rooted devices.
+No. We do not offer jailbroken or rooted devices, or penetration testing.
```

Comment:
@kristofmuhi 

What do you exactly mean with this, I assume it's not supported. If that's the case then can you change the sentence/place it in a separate question so it matches the context?